lib/ansible/modules/identity/ipa/ipa_host.py

```diff
@@ -221,7 +245,8 @@ def ensure(module, client):
                     data = {}
                     for key in diff:
                         data[key] = module_host.get(key)
-                    client.host_mod(name=name, host=data)
+                    client.host_disable(name=name)
```

Comment:
The method host_disable should be called only when a random password is requested and the ipa_host already contains 'has_keytab'=True.
Otherwise the module may perform unwanted disable operation, for instance in a scenario where only the 'description' of a host needs to be modified.

Reply:
ok